src/Hero6/Engine/Utilities/DirectoryWrapper.cs

```diff
@@ -0,0 +1,17 @@
+﻿// <copyright file="DirectoryWrapper.cs" company="Late Start Studio">
+// Copyright (C) Late Start Studio
+// This file is subject to the terms and conditions of the MIT license specified in the file
+// 'LICENSE.CODE.md', which is a part of this source code package.
+// </copyright>
+
+namespace LateStartStudio.Hero6.Engine.Utilities
+{
+    using System.IO;
+
+    public class DirectoryWrapper : IDirectoryWrapper
+    {
+        public void CreateDirectory(string path) => Directory.CreateDirectory(path);
+
+        public void Delete(string path) => File.Delete(path);
```

Comment:
I don't think this is supposed to be here, remnant from a copy paste maybe? 😉

Reply:
Oops. I was more tired than I thought. :)